apps/linkedin-to-notion/src/background/index.ts

```diff
@@ -3,14 +3,16 @@ export {};
 const linkedInURLRegex = /linkedin.com\/in\/.+/;
 
 chrome.action.onClicked.addListener((tab) => {
-  if (tab.url.match(linkedInURLRegex)) {
+  if (tab.url?.match(linkedInURLRegex)) {
```

Comment:
Tab.url cannot be undefined so for me the `?` brings no value. But feel free to share the example of what you fixed doing so 😬

Reply:
Yeah you're right. You already told me that but once again I got confused, my bad.